### PR TITLE
Pull request for Jest testing.

### PR DESCRIPTION
**Pull request by TechnoAndy**
## In this milestone, the following change were implemented.

- Used jest framework for testing.
- Created a test file ([..].test.js) for a file containing the add item and delete item functions to be tested.

### CODE REVIEW

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Jest Installed ✔️ 
- Used description() method ✔️ 


 ## Required Changes ♻️

- [ ] **_Check the comment below_**

## Optional suggestions

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/module/utilityFunctions.test.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        </div>
+    </div>
+    `;
+    const element = document.querySelector('.taskDynamic');
+    removeTask(element);
+    Tasks.remove(1);
+    expect(Tasks.TaskObject).toHaveLength(1);
+  });
+  test('test for local storage', () => {
+    JSON.parse(localStorage.getItem('TASKS_LIST'));
```
[the remark this answers is not in the document]
Comment (Daphineatim) [reply src/module/utilityFunctions.test.js]: Thank you, the issue has been fixed.

File: src/module/utilityFunctions.test.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    Tasks.add('Test');
+    document.body.innerHTML = `
+    <div id="taskList" class="h5 p-0 m-0"></div>
+    `;
+    displayContent();
+    expect(Tasks.TaskObject).toHaveLength(1);
+  });
+
+  test('test for local storage', () => {
+    JSON.parse(localStorage.getItem('TASKS_LIST'));
```
[the remark this answers is not in the document]
Comment (Daphineatim) [reply src/module/utilityFunctions.test.js]: Issue fixed.

File: src/module/utilityFunctions.test.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
+/**
```
[the remark this answers is not in the document]
Comment (Daphineatim) [reply src/module/utilityFunctions.test.js:1]: Deleted the commented code.

**Review by ichala — APPROVED** [review]
Hi Team ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Add and Delete tests are working ✔️ 
- Clean Code ✔️ 


 Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**